client/rpcserver/types.go

```diff
@@ -12,6 +12,9 @@ import (
 	"decred.org/dcrdex/dex/encode"
 )
 
+// An orderID is a 256 bit number encoded as a hex string.
+const orderIdLen = 64
```

Comment:
Let's use `hex.EncodedLen(decred.org/dcrdex/dex/order.OrderIDSize)` or just `2*order.OrderIDSize` unless importing dex/order would be a problem.